tools/stats/stats.cpp

```diff
@@ -43,8 +44,35 @@ TIP: In order to collect statistics from all .spv files under current dir use
 find . -name "*.spv" -print0 | xargs -0 -s 2000000 %s
 
 Options:
-  -h, --help                       Print this help.
-  -v, --verbose                    Print additional info to stderr.
+  -h, --help
+                   Print this help.
+
+  -v, --verbose
+                   Print additional info to stderr.
+
+  --codegen_opcode_hist
```

Comment:
I think we normally use hyphens in command line options instead of underscores.  But this is your tool so you get to choose.  :-)